frictionless/package/package.py

```diff
@@ -441,6 +441,62 @@ def to_ckan(self, target, *, control=None):
         storage.write_package(self, force=True)
         return storage
 
+    @staticmethod
+    def from_github(
```

Comment:
What if this function will just use constructor?

I think it will be the same:
```
return Package(source, control=control)
```

WDYT?

Reply:
I agree with that. Better to keep all the Catalog specific logic inside the portals/github